src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/actions/manual/ManualTriggerAction/index.jelly

```diff
@@ -132,8 +132,19 @@
                                                                    id="check${theId}"
                                                                    onClick="rowSelected('${theId}')"/>
                                                         </td>
-                                                        <td onClick="activateRow('${theId}')">
```

Comment:
I feel the number is the most common place you'd want to click for activating the row, so why remove it??